invasion/enemy.py

```diff
@@ -50,36 +50,24 @@ def __init__(self, path: str, min_enrage_mult: float, max_enrage_mult: float, en
         self.max_enrage_mult = max_enrage_mult
         self.enraged = enraged
 
-        # maybe reduce the number of these so it makes more sense which is stronger
         self._title_prefixes = [
-            "Angry",
-            "Enraged",
-            "Formidable",
-            "Rampaging",
-            "Ferocious",
-            "Epic",
-            "Mythic",
-            "Ancient",
-            "Colossal",
-            "Unstoppable",
-            "Apocalyptic",
-            "Eternal",
-            "Divine"
+            "Angry", "Enraged", "Formidable", "Rampaging", "Ferocious",
+            "Epic", "Mythic", "Ancient", "Colossal", "Unstoppable",
+            "Apocalyptic", "Eternal", "Divine"
```

Comment:
let's keep these each on a single line like they were before. makes them easier to read and maintain